boards/earlgrey-nexysvideo/src/main.rs

```diff
@@ -310,6 +311,22 @@ pub unsafe fn reset_handler() {
         static mut _sappmem: u8;
         /// End of the RAM region for app memory.
         static _eappmem: u8;
+        /// The start of the kernel stack
```

Comment:
Can we add a comment here noting that the below variables are only needed if the kernel MPU is to be used? I don't want someone to copy this board, thinking of OpenTitan as a good reference platform, and end up with this stuff they do not need.